### PR TITLE
sticky footer

### PR DESCRIPTION
**Pull request by kemalfidan**
This branch started off for the footer but I ended up finishing the admin header file and added the favicon as well.

Some screenshots:

sticky footer
Screen Shot 2021-04-15 at 5 38 56 PM
Screen Shot 2021-04-15 at 5 38 36 PM

header
Screen Shot 2021-04-16 at 5 11 30 PM
Screen Shot 2021-04-16 at 5 11 22 PM

favicon - site is cached for me on chrome so I used Safari
Screen Shot 2021-04-16 at 5 09 54 PM

### CODE REVIEW

File: src/components/Header/index.tsx [hunk 1]
```diff
@@ -1,10 +1,46 @@
...
+import Button from "@material-ui/core/Button";
 import constants from "utils/constants";
+import { useRouter } from "next/router";
+import Link from "next/link";
+import urls from "utils/urls";
+import CoreTypography from "../core/typography";
+import Menu from "@material-ui/core/Menu";
+import MenuItem from "@material-ui/core/MenuItem";
+import AccountCircleIcon from "@material-ui/icons/AccountCircle";
+import colors from "src/components/core/colors";
```
Comment (zaviermiller) [inline src/components/Header/index.tsx:13]: theres a merge conflict here, i think just overwriting with your changes will be fine. admin header no longer needed

File: src/pages/_app.tsx [hunk 2]
```diff
@@ -9,8 +9,13 @@ import CssBaseline from "@material-ui/core/CssBaseline";
 import theme from "utils/theme";
 import Header from "src/components/Header";
 import Footer from "src/components/Footer";
+import constants from "utils/constants";
+import "../../public/App.css";
+
+export default function MyApp({ Component, pageProps, router }: AppProps) {
+    let admin = false;
+    if (router.pathname.includes("admin")) admin = true;
```
Comment (zaviermiller) [inline src/pages/_app.tsx:17]: would we be able to use the auth from #51 here to check isLoggedIn or sum?

**Review by zaviermiller — APPROVED** [review]
looks good! notes arent really blocking so approved :]

File: src/pages/_app.tsx (hunk 2 above)
Comment (kemalfidan) [reply src/pages/_app.tsx:17]: I thought about that but I don't see an easy way for it. What we'll probably end up doing is have a fetch call to the `validateLogin` endpoint on every page that needs admin access. The check here is just to see if the admin header should be rendered or not.

Comment (zaviermiller) [reply src/pages/_app.tsx:17]: :+1:

**Review by andlrutt — APPROVED** [review]
👍

**Pull request merged**